app/server.py

```diff
@@ -39,15 +39,35 @@ def learnToInvest(request):
     page = app.templates.get_template('resources.html')
     return page.render()
 
-@app.route('/api/v1/saveohlvc', methods=['POST'])
-def saveOhlvc(request):
+@app.route('/dailystock/')
+def daily_stock(request):
+    """ """
+    try:
+        symbol = request.args.get('symbol')[0]
+        data = get_ohlvc(request, symbol)
+    except:
+        data = ''
+    page = app.templates.get_template('dailystock.html')
+    return page.render(ohlvc_data=data)
+
+@app.route('/api/v1/getohlvc/')
+def get_ohlvc(request, symbol):
+    """ return last 10 days of recent daily stock data for a symbol """
+    recent_ohlvc_data = db.get_recent_ohlvc(symbol)
+    return recent_ohlvc_data
+
+@app.route('/api/v1/saveohlvc/', methods=['POST'])
+def update_ohlvc(request):
     """ get and save all recent daily stock data for a symbol """
     symbol = request.args.get('symbol')[0]
-    start_date = db.get_start_date(symbol)
-    ohlvc_data = quandl.get("YAHOO/{}".format(symbol), start_date=start_date)
-    db.save_stock_data(ohlvc_data, symbol)
-    return 'ok'
-
+    if db.symbol_exists(symbol):
```

Comment:
good catch. yeah i was checking if the symbol was in the db, but you are right... I actually want to make an API call to get the data if the symbol is not in the db.  I will update this.